runtime/ggma/examples/generate_text/requirements.txt

```diff
@@ -0,0 +1 @@
+transformers==4.50.3
```

Comment:
tico==0.1.0.dev251123

`pip install tico==0.1.0.dev251123` will install nightly version.
Uninstall pre-existing tico version first before running above command. As it's nightly, version manager will assume that it's earlier than the 0.1.0 version not to upgrade. 

@glistening